src/main/cli/cli.c

```diff
@@ -5426,11 +5470,17 @@ const clicmd_t cmdTable[] = {
 #endif
 #endif
     CLI_COMMAND_DEF("name", "name of craft", NULL, cliName),
+#ifdef USE_OSD
+    CLI_COMMAND_DEF("profile_name", "name of pid profile", NULL, cliPidProfileName),
+#endif
```

Comment:
The commands need to be defined in alphabetical order otherwise this breaks the tab auto-complete.

Reply:
Thanks - fixed.